bin/sidecli.re

```diff
@@ -265,9 +267,9 @@ let info_withdraw = {
 let withdraw =
     (folder_node, sender_wallet_file, tezos_address, amount, ticket) => {
   open Networking;
-  let.await validators_uris = validators_uris(folder_node);
-  let validator_uri = List.hd(validators_uris);
-  let.await block_level_response = request_block_level((), validator_uri);
+  let file = folder_node ++ "/identity.json";
+  let.await identity = Files.Identity.read(~file);
```

Comment:
it's not really in the scope of the PR but at some point we should write a function that combines these (and also uses ` Filename.concat`)